README.md

```diff
@@ -0,0 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CLONED_NYT
+
+> A clone of New York Times article
+
+![screenshot](Assets\ProjectScreenShot.jpg)
```

Comment:
The image path is wrong, please fix it. You can update from `![screenshot](Assets\ProjectScreenShot.jpg)` to `![screenshot](Assets/ProjectScreenShot.jpg)`